packages/jest/test/__snapshots__/react-enzyme.test.js.snap

```diff
@@ -114,6 +114,33 @@ exports[`enzyme mount theming 1`] = `
 </div>
 `;
 
+exports[`enzyme mount using the css helper 1`] = `
+.emotion-0 {
+  display: -webkit-box;
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display: flex;
+}
+
+.emotion-3 {
+  background-color: black;
+  color: white;
+}
+
+<div
+  className="emotion-0"
+>
+  Test content
+  <div
+    className="emotion-1 emotion-2"
```

Comment:
need to look at why this extra class name is present

Reply:
It seems that you have already resolved this issue, right?